avm/ptn/lz/sub-vending/tests/e2e/hub-spoke/main.test.bicep

```diff
@@ -0,0 +1,79 @@
+metadata name = 'Hub and spoke topology.'
+metadata description = 'This instance deploys a subscription with a hub-spoke network topology.'
+
+targetScope = 'managementGroup'
+
+@description('Optional. The location to deploy resources to.')
+param resourceLocation string = 'uksouth'
+
+@description('Optional. The subscription billing scope.')
+param subscriptionBillingScope string = 'providers/Microsoft.Billing/billingAccounts/7690848/enrollmentAccounts/350580'
+
+@description('Optional. A token to inject into the name of each resource.')
+param namePrefix string = '#_namePrefix_#'
+
+@description('Optional. A short identifier for the kind of deployment. Should be kept short to not run into resource-name length-constraints.')
+param serviceShort string = 'ssahs'
+
+@description('Optional. A short guid for the subscription name.')
+param subscriptionGuid string = toLower(substring(newGuid(), 0, 3))
+
+module nestedDependencies 'dependencies.bicep' = {
+  name: '${uniqueString(deployment().name, resourceLocation)}-nestedDependencies'
+  scope: resourceGroup('9948cae8-8c7c-4f5f-81c1-c53317cab23d', 'rsg-blzv-perm-hubs-001')
```

Comment:
That ID looks veeery tenant-specific. If I'd want tog et this running in my tenant I wouldn't even know I need to update something here. I'd strongly recommend to add a comment here and call out what a consumer would need to do to get this running themselves

Reply:
Same, btw with the billing account

Reply:
And the same for the other test files. Won't add extra comments :)